examples/community/README.md

```diff
@@ -1130,3 +1130,77 @@ Init Image
 Output Image
 
 ![img2img_clip_guidance](https://huggingface.co/datasets/njindal/images/resolve/main/clip_guided_img2img.jpg)
+
+
+### Stable Diffusion on IPEX
+
+This diffusion pipeline can accelarate the inference of Stable-Diffusion on Intel CPUs with BF16/FP32 precision by [IPEX](https://github.com/intel/intel-extension-for-pytorch).
+
+To use this pipeline, You need to:
+1. Install [IPEX](https://github.com/intel/intel-extension-for-pytorch)
+```python
+python -m pip install intel_extension_for_pytorch
+```
```

Comment:
My understanding is that IPEX requires specific versions of PyTorch (1.13.1?), so I'd also mention it here, particularly because PyTorch 2 is now the default and more performant in other platforms.

Reply:
Pytorch/IPEX2.0 is working. It is recommended to use Pytorch/IPEX2.0 to get the best performance boost of flash attention optimization. It can also work on Pytorch/IPEX 1.13. I have added the information about the version.